script.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] Kindly don't disable this linter because you are only allowed to disable it for no-used-vars and max-classes but apart from these you cannot disable linter for anything else :thinking: because linters check your javascript files and verify if their code is syntactically correct :+1: Also, it prevents you from implicitly introducing those errors in your code which can break your whole application :blush:

**Solution**
You can use `+= 1` instead of `++` in order to fix the linter error :+1:

Reply:
Done @HAMMAS-SALEEM, I've already made the change.